### DISCUSSION TITLE
2023-07-13

### DISCUSSION THREAD

**Discussion by aak2075** [woowacourse-teams/2023-haru-study#22] [daily_meeting]
### 오늘의 할 일
- 공통
    - api 명세 픽스
    - 세부정책 포함
    - ci/cd 구축(마코, 엽토)

- 백엔드
    - 기능목록 작성(테오, 모디)

- 프론트엔드
    - 디자인(룩소, 노아)

### 내일의 할 일
- 공통
    - 뽀모도로 템플릿 질문 확정(노아, 룩소)
    - 세부 정책
    - 인스턴스 생성 및 cd 구축(엽토, 마코)
- 각자
    - 맡은 부분 이슈 발행

### 언젠가 해야할 일
- 기획 변경된 이유 정리

**Comment by yeopto** [comment]
### 체크인

4점 - 머리가 너무 아프네요.. 컨디션이 안좋습니다.

### 체크이읏

7점 - 좀 나아져서 +3

**Comment by nlom0218** [comment]
### 체크인

10점 - 버스를 놓쳐 그 다음 버스를 탔지만 다행히 지각은 면해서 다행이다

**Comment by woosung1223** [comment]
### 체크인
6점 - 어제 이불 빨래한다고 2시에 잤더니 졸립네여 그런데 이불이 뽀송해서 좋긴함

### 체크아웃
8점 - 오늘 데이터베이스 설계를 해서 기부니가 좋아요

**Comment by aak2075** [comment]
### 체크인

9점 - 출근길이 험난해서(-1)

**Comment by jaehee329** [comment]
### 체크인
8점 - 오는 길에 습덥해서 땀이 난 것 빼곤 괜찮다

### 체크아웃
9점 - 슬슬 개발의 향기가 난다.... 익숙하다...

**Comment by woo-jk** [comment]
### 체크인

8점 - 애매할 땐 버스를 타지말자🥲

### 체크아웃

9점 - 디자인이 끝났고 이제 개발 들어갈 수 있다😊
